src/workerd/server/server.c++

```diff
@@ -2033,6 +2042,18 @@ kj::Own<Server::Service> Server::makeWorker(kj::StringPtr name, config::Worker::
       }
     }
 
+    kj::HashMap<kj::StringPtr, WorkerService::ActorNamespace&> durableNamespacesByUniqueKey;
+    for(auto& [className, ns] : workerService.getActorNamespaces()) {
+      KJ_IF_MAYBE(config, ns.getConfig().tryGet<Server::Durable>()) {
+        auto& actorNs = ns; // clangd gets confused trying to use ns directly in the capture below??
```

Comment:
Yeah, it's the structured binding above. Basically, you don't have a name for `ns` because it comes out of a temporary. Imagine what you'd have to do if it was `auto& nsPair` instead.